From 19b89c9459b8bafa0fb7ce5d23eb00121e5100ed Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Tue, 3 Mar 2015 23:38:16 +0100
Subject: [PATCH 2/2] Set up a conflict with standard ROS pkg and other OSRF
 wrappers

---
 debian/control.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control.em b/debian/control.em
index a7f88e0..cba26f2 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -11,4 +11,5 @@ Standards-Version: 3.9.2
 Package: @(Package.replace('gazebo-','gazebo5-'))
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Conflicts: @(Package), @(Package)-current, @(Package.replace('gazebo-','gazebo3-')), @(Package.replace('gazebo-','gazebo4-'))
 Description: @(Description)
-- 
1.9.1

